Pipfile

```diff
@@ -59,7 +59,7 @@ werkzeug = "~=2.1.1"
 prometheus-client = "==0.14.1"
 gds-metrics = {version = "==0.2.4", ref = "6f1840a57b6fb1ee40b7e84f2f18ec229de8aa72", git = "https://github.com/alphagov/gds_metrics_python.git"}
 packaging = "==21.3"
-notifications-utils = {editable = true, branch = "main", git = "https://github.com/GSA/notifications-utils"}
+notifications-utils = {editable = true, ref = "main", git = "https://github.com/GSA/notifications-utils"}
```

Comment:
Not sure why I thought the other way was right, but you're right and I can't find more info in the `pipenv` docs. While we're fixing it, the URL should end in `.git`

Reply:
I wasn't sure if this was any more right, but it's what got output when I used `pipenv install` on the CLI to update the version so I assumed it was fine. I think the `.git` doesn't matter (things are getting installed, anyway), but I'll add it in here and on the admin side

Reply:
Yep, it definitely seems like it works both ways. I tried to find my way through docs and searches to any sort of explanation of the options/requirements, and came up empty, so 🤷🏻 .